src/LoginForm/LoginForm.js

```diff
@@ -29,25 +30,29 @@ export class LoginForm extends Component {
 }
 
   verifyCredentials = (e) => {
-    e.preventDefault();
-    if (!this.state.userName && !this.state.password) {
+    // e.preventDefault();
+    if (!this.state.userName && !this.state.password && !this.state.email) {
```

Comment:
we need to display a visual message if the information is not correct

Reply:
Gotcha! We should probably make an issue for this. We could maybe start with an alert and make it more UX/UI friendly from there.